apps/system/lockscreen/js/lockscreen_inputpad.js

```diff
@@ -66,7 +70,7 @@
   LockScreenInputpad.prototype.renderUI = function() {
     return new Promise((resolve, reject) => {
       this.toggleEmergencyButton();
-      this.updatePassCodeUI();
+      this.setPinLength(this.pinLength);
```

Comment:
But since the `setPinLength` conditionally call the `updatePassCodeUI`, the condition may need to be moved to here as well, rather than in the setter.